jenkins-worker/Worker_Dockerfile

```diff
@@ -0,0 +1,78 @@
+  # Based off of evarga/jenkins-slave
+# This Dockerfile is used to build an image containing basic stuff to be used as a Jenkins slave build node.
+FROM ubuntu:trusty
+
+# This was the original, trying to make it match the frontend version
+#FROM ubuntu:14.04
+# Original mainainer: Matt Shanker
+MAINTAINER engineering@socrata.com
+
+# Add lines in limits.conf to make the ulimit higher, default of 1024 is from the stone age
+RUN echo '*  	hard	    nofile  	525488' >> /etc/security/limits.conf
+RUN echo '*  	soft	    nofile  	525488' >> /etc/security/limits.conf
+
+# Add lines to the locale related files
+RUN echo "LC_ALL=en_US.UTF-8" >> /etc/environment
+RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+RUN echo "LANG=en_US.UTF-8" > /etc/locale.conf
+RUN locale-gen en_US.UTF-8
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+RUN update-locale en_US.UTF-8
+
+# Make sure the package repository is up to date, upgrade out of date packages.
+# Add python-software-properties, which installs the `add-apt-repository` script
+# Enable a new repo for git, the default for Ubuntu 10.04 is way too old
```

Comment:
Just a curiosity question. Since this comment makes reference to the 10.04 way as "too old", I wonder if the 18.04 way is just right now?

Reply:
updated to say 14.04 I think that was a typo on Matt's part.